partner_create_by_vat/models/res_partner.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# ©  2015 Forest and Biomass Services Romania
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, api, _
+from openerp.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def _get_vies_data(self, vat):
+        try:
+            from stdnum.eu.vat import check_vies
```

Comment:
Use the guideline of [external-dependencies](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#external-dependencies)
```python
try:
   from stdnum.eu.vat import check_vies
except ImportError:
      _logger.debug('Cannot `import external_dependency_python_N`.')
```

And consider the guideline of [import-order](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports)